server/server.go

```diff
@@ -3090,13 +3484,23 @@ func (s *StanServer) processSubscriptionRequest(m *nats.Msg) {
 		}
 	}
 
+	// In clustered mode, drop the request if we are not the channel leader.
```

Comment:
I guess this was meant to be done below

Reply:
Oops, mistake. This comment should be removed.